CONTRIBUTING.md

```diff
@@ -217,14 +217,20 @@ flutter test test/ --update-goldens
 
 To run master version of DevTools with all experimental features enabled:
 
-1. Start DevTools
+1. Make sure your Flutter is on the channel `master`:
+
+```
+flutter channel master
```

Comment:
Instead of switching to master, we should run `./tool/update_flutter_sdk.sh --local`

Reply:
It does not work for me:
```
polinach-macbookpro3% ./tool/update_flutter_sdk.sh --local
STATUS: Updating local Flutter checkout to version 'HttpStatus 401: Unauthorized'.
~/_/platform/polina_flutter ~/_/devtools
error: pathspec 'HttpStatus' did not match any file(s) known to git
error: pathspec '401:' did not match any file(s) known to git
error: pathspec 'Unauthorized' did not match any file(s) known to git
Flutter 3.5.0-3.0.pre • channel unknown • unknown source
Framework • revision 529184bcca (3 days ago) • 2022-10-07 10:28:49 -0700
Engine • revision 7de65af6f8
Tools • Dart 2.19.0 (build 2.19.0-288.0.dev) • DevTools 2.18.0
~/_/devtools
STATUS: Finished updating local Flutter checkout.
STATUS: Updating 'tool/flutter-sdk' to version 'HttpStatus 401: Unauthorized'.
STATUS: 'tool/flutter-sdk' directory does not exist - cloning it now
fatal: destination path 'flutter-sdk' already exists and is not an empty directory.
~/_/devtools/flutter-sdk ~/_/devtools
error: pathspec 'HttpStatus' did not match any file(s) known to git
error: pathspec '401:' did not match any file(s) known to git
error: pathspec 'Unauthorized' did not match any file(s) known to git

┌─────────────────────────────────────────────────────────┐
│ A new version of Flutter is available!                  │
│                                                         │
│ To update to the latest version, run "flutter upgrade". │
└─────────────────────────────────────────────────────────┘
Flutter 3.5.0-4.0.pre.10 • channel master • https://github.com/flutter/flutter
Framework • revision 92e9d57acf (3 days ago) • 2022-10-07 18:43:52 -0400
Engine • revision 257db6c0e0
Tools • Dart 2.19.0 (build 2.19.0-291.0.dev) • DevTools 2.18.0
~/_/devtools
STATUS: Finished updating 'tool/flutter-sdk'.
```